mmdet3d/models/losses/__init__.py

```diff
@@ -10,5 +11,5 @@
     'FocalLoss', 'SmoothL1Loss', 'binary_cross_entropy', 'ChamferDistance',
     'chamfer_distance', 'axis_aligned_iou_loss', 'AxisAlignedIoULoss',
     'PAConvRegularizationLoss', 'UncertainL1Loss', 'UncertainSmoothL1Loss',
-    'MultiBinLoss'
+    'MultiBinLoss', 'weighted_sigmoid_focal_loss', 'weighted_smoothl1'
```

Comment:
Is it necessary to add these two new losses? We should have supported similar ones in either mmdet or mmdet3d?